x-pack/solutions/observability/plugins/profiling/public/routing/index.tsx

```diff
@@ -247,7 +233,7 @@ const routes = {
                 element: (
                   <RouteBreadcrumb
                     title={i18n.translate('xpack.profiling.breadcrumb.differentialFunctions', {
-                      defaultMessage: 'Differential Top N',
+                      defaultMessage: 'Differential top N functions',
```

Comment:
Similar here -> also the t is capital in the title: 
<img width="635" alt="image" src="https://github.com/user-attachments/assets/abde950f-3a9d-472a-baf7-4efb80a1490b" />